examples/binary/cloudbuild/BuildEventData-complex.json

```diff
@@ -0,0 +1,75 @@
+{
+  "timeout": "660s",
+  "createTime": "1993-07-25T02:32:41.388Z",
+  "tags": [
+    "dolor exercitation",
+    "sit id consequat",
+    "dolore eu sit irure esse"
+  ],
+  "buildTriggerId": "cupidatat",
+  "logUrl": "http://example.com",
+  "images": [
+    "http://example.com",
+    "http://example.com"
+  ],
+  "options": {
+    "substitutionOption": "MUST_MATCH",
+    "env": [
+      "string1"
+    ],
+    "volumes": [],
+    "machineType": 1,
+    "logging": "GCS_ONLY",
+    "diskSizeGb": 18311588,
+    "sourceProvenanceHash": [
+      "SHA256",
+      "MD5"
+    ],
+    "secretEnv": [
+      "secret"
+    ],
+    "logStreamingOption": "STREAM_ON",
+    "workerPool": "cupidatat",
+    "requestedVerifyOption": "NOT_VERIFIED"
+  },
+  "results": {
+    "buildStepOutputs": [
+      "out1"
+    ],
+    "images": [],
+    "numArtifacts": 3306780,
+    "buildStepImages": [
+      "http://example.com"
+    ],
+    "artifactTiming": {
+      "startTime": "1995-03-02T21:38:07.876Z",
+      "endTime": "2006-06-09T14:32:38.775Z"
+    },
+    "artifactManifest": "officia aliqua mollit"
+  },
+  "artifacts": {
+    "images": [
+      "http://example.com"
+    ],
+    "objects": {
+      "location": "laborum sunt in",
+      "paths": []
+    }
+  },
+  "status": "SUCCESS",
+  "secrets": [
+    {
+      "kmsKeyName": "ut magna velit"
+    }
+  ],
+  "source": {
+    "storageSource": {
+      "bucket": "sample-bucket",
+      "object": "my-object.txt",
+      "generation": "1587627537231057"
+    }
+  },
+  "logsBucket": "esse sunt ex",
```

Comment:
I doubt that that's a valid bucket ID - likewise the project ID beneath it. (I know that's what's in the testdata one.)

Given that this is for customers to read, I think it would be making sure all the values are as realistic as possible, without including any genuine PII of course.